README.md

```diff
@@ -109,6 +109,22 @@ env:
   value: *containerName # referencing earlier anchor in same yaml
 ```
 
+## DICOM Redactor
+
+Import adapter can be configured to use [DICOM Redactor Library](https://github.com/GoogleCloudPlatform/healthcare-deid/tree/master/offline_tools/redactorhttps://github.com/GoogleCloudPlatform/healthcare-deid/tree/master/offline_tools/redactor) to redact sensitive data contained in DICOM tags during C-STORE upload.
```

Comment:
Phrasing edit: "The import adapter can be configured to use the [DICOM Redactor Library]"
... "during a C-STORE upload."

Reply:
Ok